conjure-codegen/src/servers.rs

```diff
@@ -449,11 +449,12 @@ fn generate_endpoint_impl(
 
     let parts = quote!(parts_);
     let body = quote!(body_);
-    let safe_params = if has_safe_params(ctx, endpoint) {
-        quote!(safe_params_)
+    let response_extensions = if has_safe_params(ctx, endpoint) {
+        quote!(response_extensions_)
```

Comment:
Is there a test that hits this code path? Looks like the generated test services don't have safe params.